AppCreationScripts/Apps.json

```diff
@@ -42,13 +42,14 @@
       ],
      "codeConfigurations": [	
        {
-		     "settingFile": "/src/main/resources/application.properties", 
-		     "replaceTokens": 
-           {
-             "appId": "Enter_the_Application_Id_here",
-             "tenantId": "Enter_the_Tenant_Info_Here",
-             "clientSecret": "Enter_the_Client_Secret_Here"
-			     }
+         "settingFile": "/src/main/resources/application.properties",
+         "replaceTokens": {
+           "appId": "Enter_the_Application_Id_here",
+           "tenantId": "Enter_the_Tenant_Info_Here",
+           "clientSecret": "Enter_the_Client_Secret_Here",
+           "authorityEndpointHost": "https://login.microsoftonline.com/",
+           "msgraphEndpointHost": "https://graph.microsoft.com/"
```

Comment:
I don't see the value "https://graph.microsoft.com/" corresponding to the key "msgraphEndpointHost". The Azure portal will fail to replace this because it won't find anything to replace for msgraphEndpointHost in application.properties file.

Reply:
@archieag Thanks for catching this. I have update application.properties